sm-shop/src/test/java/com/salesmanager/shop/admin/controller/categories/CategoryControllerTest.java

```diff
@@ -20,6 +21,7 @@
 
 @SpringBootTest(classes = ShopApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @RunWith(SpringRunner.class)
+@Ignore
```

Comment:
what's wrong with this test class to ignore? 
would you mind adding a msg with the reason as a param in the annotation so we can fix it in the future?

Reply:
It's just an empty test class for the category controller, that we decided to postpone it's implementation, till the release of the new admin portal